### DISCUSSION TITLE
Disable/hide internalLink in Link Map

### DISCUSSION THREAD

**Discussion by Spheriax** [TriliumNext/Trilium#2414] [Q&A]
Hi!

I've been trying out Trilium for a while and I'm really enjoying it. 
I especially like the ability to add relationship attributes and the subsequent visualization by Link Map. 
However, if I create a internal linke in Note A to note B, the `internalLink` attribute shows up.

![demolink](https://user-images.githubusercontent.com/88401173/144886390-821d2c0b-ae6a-41df-98b8-68925b2d9b40.png)

Neither note has `internalLink` listed as attribute.
Is there a way to disable or hide the `internalLink` attribute from the Link Map?
The wiki shows a screenshot with a checkbox but I can't for the live of me find that. 

Thanks in advance!

**Comment by zadam** [comment]
Hi, that screenshot is outdated and it's currently not possible to configure which relations are visible.

`internalLink` is not show in attributes since it's automatically managed by trilium and not really user editable through attributes interface. But it does exist and it is relevant in the note map since the notes are linked together ...
